packages/wrangler/src/d1/execute.tsx

```diff
@@ -52,7 +51,7 @@ type QueryResult = {
 	query?: string;
 };
 // Max number of bytes to send in a single /execute call
-const QUERY_LIMIT = 1_000_000; // 1MB
+const QUERY_LIMIT = 10_000;
```

Comment:
What is the benefit in sending 10kb over 1MB?

Reply:
QUERY_LIMIT is actually the number of SQL statements going into a single D1 API request inside a transaction, not MB. 10,000 is a reasonable number that's efficient/fast enough when importing big SQL files, but also doesn't kill D1's DO from resource exhaustion.